.github/checkgroup.yml

```diff
@@ -212,6 +209,8 @@ subprojects:
 
   - id: "lightning_lite: Tests"
     paths:
+      - "examples/lite/**"
+      - "examples/run_lite_examples.sh"
```

Comment:
I think these additions should be moved to L236 with the lite Azure GPU group

Reply:
Got it!